docs/IgorFromUA/index.html

```diff
@@ -0,0 +1,104 @@
+                <img class="header__logo" src="./src/logo.png" width="55" height="58" alt="logo masters academy">
+                <nav class="navigation">
+                    <div class="navigation__menu-icon">
+                        <svg width="34" height="19" viewBox="0 0 34 19">
+                            <rect width="34" height="3.5" rx="1" fill="#FFF"></rect>
+                            <rect y="8" width="34" height="3.5" rx="1" fill="#FFF"></rect>
+                            <rect y="16" width="34" height="3.5" rx="1" fill="#FFF"></rect>
+                        </svg>
+                    </div>
+                    <ul class="navigation__list">
+                        <li class="navigation__item">
+                            <a class="navigation__link" href="./calculator.html">Calculator</a>
+                        </li>
+                        <li class="navigation__item">
+                            <a class="navigation__link" href="./games.html">Games</a>
+                        </li>
+                        <li class="navigation__item">
+                            <a class="navigation__link navigation__link_active" href="#">About</a>
+                        </li>
+                        <li class="navigation__item">
+                            <a class="navigation__link" href="./login.html">Login</a>
+                        </li>
+                    </ul>
+                </nav>
+            </header>
+            <main class="container">
+                <div class="wrapper__about">
+                    <img class="about__image" src="./src/image.png" width="532" height="356" alt="work space">
+                    <section class="about">
+                        <h1 class="about__title">About me</h1>
+                        <p class="about__text">Hi! My name is Ihor and I'm a Junior Frontend Developer. I am already familiar
+                            with
+                            main
+                            Web Technologies like HTML, CSS, JavaScript and Git version control system. </p>
+                        <p class="about__text">
+                            This page was developed during the course <a class="about__link" target="_blank"
+                                href="https://www.mastersacademy.education/frontend-for-beginners-it">'Frontend
+                                for beginners' from Masters Academy in 2023.</a>
+                        </p>
+                        <p class="about__text">
+                            This is a social project from MOCG company where I got an opportunity to work with Frontend mentors
+                            and
+                            to
+                            create my own small project for the portfolio.</p>
+                        <p class="about__text">
+                            You can contact me via <a href="https://www.linkedin.com/in/ihor-zapisochnyi-751b9424b/" target="_blank">Linkedin</a>
```

Comment:
I think, all links of your project should have similar color
Also, please correct the font-size of header links according to design

![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/ae8d9246-aae3-41c3-a73a-f611c57c1f41)